app/models/proposal/token.rb

```diff
@@ -62,7 +62,7 @@ def validate_accepted
     scope :reminded, ->{ where('reminded_at IS NOT NULL') }
 
     def expects
-      @expects || proposable.proposal_options[:expects]
+      @expects ||= proposable.proposal_options[:expects]
```

Comment:
What is the purpose of this operation? did you mean to 'cache with instance variable' here? If yes, then I'd rather use that conditional assignment here.

Just to clarify I grepped for that instance variable name

```bash
(rails-5) proposal: grep -Rin "@expects" *
app/models/proposal/token.rb:65:      @expects ||= proposable.proposal_options[:expects]
```
```@expects``` is not used anywhere else, so this suggests it's only used for caching in #expects method.